### DISCUSSION TITLE
Error when using Firebase cloud messaging on Edge browser

### DISCUSSION THREAD

**Discussion by spl-dev1** [firebase/flutterfire#9585] [Q&A]
Just run the example in Android Studio straightly from the firebase_messaging package. All are well with Android simulators and Chrome browser. That is, a message will be received each time the floating action button is clicked. But when it comes to Edge browser, there is no messages at all. Besides, it shows a request error (status code 500) in the console. So is there anything I've missed? Please advise. Thanks.

**Comment by Ehesp** [comment]
Could you show the error message?

**Reply by spl-dev1** [reply]
Hello Ehesp,

Firstly, this is the console output of Android Studio:

![image](https://user-images.githubusercontent.com/102708819/191639382-76c00a57-ec95-41d6-a62b-fb2c8a4ac601.png)

Then, the followings are from the developer console of Edge browser:

Console:

![image](https://user-images.githubusercontent.com/102708819/191639676-22435381-ece5-4567-880d-5f17f96cc756.png)

Network:

![image](https://user-images.githubusercontent.com/102708819/191639857-854ad9e6-2faa-4c93-a541-86de204eb4b7.png)

![image](https://user-images.githubusercontent.com/102708819/191639909-afe65391-5d98-4750-8fad-8ffa38c14ddd.png)

![image](https://user-images.githubusercontent.com/102708819/191639969-12547203-9316-4e3d-a031-6bddd9bd66e5.png)



Thanks.
